README.md

```diff
@@ -94,6 +94,16 @@ if (shell.exec('git commit -am "Auto-commit"').code !== 0) {
 }
 ```
 
+## Exclude options
+
+If you need to pass a parameter that looks like an option, you can do so like:
+
+```js
+shell.grep('--', '-v', 'path/to/file'); // Search for "-v", no grep options
```

Comment:
Hmmm... What if I want to `grep` recursively for `'-v'`:
```js
shell.grep('-r', '-v', 'path/to/dir');
```
In this case, `'-v'` will be treated as an option, which I don't want. So, using `'--'`:
```js
shell.grep('--', '-r', '-v', 'path/to/dir');
```
But now `'-r'` is not treated as an option, which I also don't want.

Reply:
I guess, in the case `grep` specifically, this is how you do the above:
```sh
$ grep -r -- -v path/to/dir
```
So, `--` is telling `grep` to take the next argument literally, as opposed to taking all arguments literally.

Reply:
No, see the added docs. The correct way to do what you describe is `grep('-r', '-v', 'path/to/dir');`

Reply:
Of course. I forgot that all arguments must be passed at once.